OasisDEX/oasis-borrow
# Lw/5888 follow unfollow vault

# [Follow/Unfollow ui component](https://app.shortcut.com/oazo-apps/story/5888/vault-overview-follow-unfollow-vault)
  
## Changes 👷‍♀️
- Created follow button and it's control
  
## How to test 🧪
 - Add `      <FollowButtonControl />` for example in HomepageView.tsx before last </Flex>

- Expect initial state to be Loading
![image](https://user-images.githubusercontent.com/6871923/203549320-b18d512e-e6b2-4e8e-adef-86322ccddf99.png)
- After loading is finished click button, expect loading again, changing state from follow to unfollow
- 
![image](https://user-images.githubusercontent.com/6871923/203549751-71ab4a62-38a7-4b86-a0fb-67f5a98a479a.png)
